Gemfile

```diff
@@ -13,7 +13,7 @@ group :development do
   gem 'komenda', '~> 0.1.6'
   gem 'launchy'
   gem 'rake', '10.4.2'
-  gem 'rubocop'
+  gem 'rubocop', '~> 0.43.0'
```

Comment:
Can you explain this change? How come that when I started from scratch as well, my Rubocop run worked, whereas on CI I got the errors? Are you saying different versions are getting installed depending on the environment?